tools/cellprofiler/image_math.xml

```diff
@@ -0,0 +1,301 @@
+<tool id="image_math" name="ImageMath" version="0.1.0" python_template_version="3.5">
+    <description>perform simple mathematical operations on image intensities in CellProfiler</description>
+    <macros>
+        <import>macros.xml</import>
+        <xml name="select_category" token_order="first">
```

Comment:
this one is only used once? Is there no second-order?

Reply:
used at least once but in most of the cases, twice. Is `token_order="first"` not appropriate here?